Lib/fontTools/feaLib/builder.py

```diff
@@ -8,15 +8,19 @@
 import itertools
 
 
-def addOpenTypeFeatures(featurefile_path, font):
-    builder = Builder(featurefile_path, font)
+def addOpenTypeFeatures(font, featurefile_path=None, featurefile_data=None):
```

Comment:
I suggest to rename the keyword arguments simply "path" and "data" as it's shorter and it's clear what it is about